cdc/sink/dispatcher/partition/ts.go

```diff
@@ -11,20 +11,23 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package dispatcher
+package partition
 
 import "github.com/pingcap/tiflow/cdc/model"
 
-type tsDispatcher struct {
+// TsDispatcher is a partition dispatcher which dispatch events based on ts.
+type TsDispatcher struct {
 	partitionNum int32
 }
 
-func newTsDispatcher(partitionNum int32) *tsDispatcher {
-	return &tsDispatcher{
+// NewTsDispatcher creates a TsDispatcher.
+func NewTsDispatcher(partitionNum int32) *TsDispatcher {
+	return &TsDispatcher{
 		partitionNum: partitionNum,
 	}
 }
 
-func (t *tsDispatcher) Dispatch(row *model.RowChangedEvent) int32 {
+// DispatchRowChangedEvent returns the target partition to which a row changed event should be dispatched.
```

Comment:
Please try to keep line width within 80 chars.

Reply:
Fixed.